src/react-native/xcode.ts

```diff
@@ -45,7 +53,15 @@ export function patchBundlePhase(
   }
 
   const script: string = JSON.parse(bundlePhase.shellScript);
-  bundlePhase.shellScript = JSON.stringify(patch(script));
+  const patchedScript = patch(script);
+  if (patchedScript instanceof ErrorPatchSnippet) {
+    await showCopyPasteInstructions(
```

Comment:
I think we should clearly state they need to update the existing code. 

To avoid the user just copy pasting the shown code at the end of the build script.

The snippet method already allows to color the new/removed/existing code.

Reply:
Good point @krystofwoldrich 👍 
Updated with 94f44edf675d63abab39771da3f6db22adfebaeb